scripts/build/Dockerfile.archlinux

```diff
@@ -5,40 +5,11 @@ ARG CC=gcc
 # Initialize machine ID
 RUN systemd-machine-id-setup
 
-RUN pacman -Syu --noconfirm \
-	$CC \
-	bash \
-	make \
-	coreutils \
-	git \
-	gnutls \
-	libaio \
-	libcap \
-	libnet \
-	libnl \
-	nftables \
-	pkgconfig \
-	protobuf-c \
-	protobuf \
-	python-pip \
-	python-protobuf \
-	which \
-	sudo \
-	iptables \
-	nftables \
-	iproute2 \
-	tar \
-	bash \
-	go \
-	python-yaml \
-	asciidoctor \
-	python-importlib-metadata \
-	libdrm \
-	util-linux-libs \
-	diffutils
+RUN pacman -Syu --noconfirm $CC
 
 COPY . /criu
 WORKDIR /criu
+RUN pacman -Syu --noconfirm $(sed 's/\#.*$//' contrib/pacman-packages.txt)
```

Comment:
Would it be possible to create subdirectory called "dependencies" and add "pacman.txt", "apk.txt", "apt.txt" inside?